views/LocationListView/LocationListView.tsx

```diff
@@ -4,16 +4,33 @@ import { Button, ListItem, Typography } from '../../.storybook/stories';
 import { LayoutProvider, NavbarWithLogout } from '../../components';
 import { LoadingWrapper } from '../../components/LoadingWrapper';
 import { useAppNavigation } from '../../hooks';
-import { resolveUserID } from '../../utils';
-import { BOT_NAMES } from '../../utils/env';
+import {
+  ReadStoredValue,
```

Comment:
[general comment] I'm not too fond of the flow of the current solution as it uses storage to save data about bots and requires clicking a button to start loading:
1. Add a unique bot prefix to distinguish user's bots from our HOME bots
2. Instead of loading the bot's list on the button click, load it on the view init (use `useFocusEffect`)
3. use `useListenForBotFather` instead of creating a new function